serial_views.py

```diff
@@ -25,10 +33,15 @@
     ZERO: 60,
 }
 
+# provide newest dataprocessing id
+EGO_DP_VERSION = settings.config['WAM_APP_FRED']['EGO_DP_VERSION']
```

Comment:
I am not sure this should be defined in the config.cfg of wam core

Reply:
Thats what I wanted to know before merging. I still think it make sence to outsource this from the code. I will use the app.cfg I already created 3 months ago :) maybe rename it fred_app.cfg even better @Bachibouzouk